packages/arb-token-bridge-ui/src/components/TransferPanel/TransferDisabledDialog.tsx

```diff
@@ -14,6 +14,70 @@ import { useSelectedTokenIsWithdrawOnly } from './hooks/useSelectedTokenIsWithdr
 import { isTransferDisabledToken } from '../../util/TokenTransferDisabledUtils'
 import { isTeleportEnabledToken } from '../../util/TokenTeleportEnabledUtils'
 import { addressesEqual } from '../../util/AddressUtils'
+import { isValidLifiTransfer } from '../../pages/api/crosschain-transfers/utils'
+import { ERC20BridgeToken } from '../../hooks/arbTokenBridge.types'
+import { isLifiEnabled } from '../../util/featureFlag'
+import { CommonAddress } from '../../util/CommonAddressUtils'
+
+export function isDisabledCanonicalTransfer({
```

Comment:
Can there be some reuse in logic of `isArbitrumCanonicalTransfer()` function defined in another place?

Reply:
Discussed offline. `isDisabledCanonicalTransfer` is called by `isArbitrumCanonicalTransfer`